R/layers.R

```diff
@@ -76,20 +87,43 @@
 #'
 #' output_path <- tempfile(fileext = ".css")
 #' sass(layer, output = output_path, write_attachments = TRUE)
-#' @name sass_layer
-sass_layer_merge <- function(...) {
-  layers <- dropNulls(rlang::list2(...))
-  is_layer <- vapply(layers, is_sass_layer, logical(1))
-  layers[!is_layer] <- lapply(layers[!is_layer], function(x) {
-    sass_layer(rules = x)
-  })
-  Reduce(sass_layers_join, layers)
+#' @describeIn sass_layer Compose the parts of a single Sass layer. Object returned is a `sass_bundle()` with a single Sass layer
+#' @export
+sass_layer <- function(
+  defaults = NULL,
+  declarations = NULL,
+  rules = NULL,
+  html_deps = NULL,
+  file_attachments = character(0),
+  tags = NULL
+) {
+
+  if (!missing(tags)) {
+    .Deprecated(msg="`sass_layer(tags)` is deprecated. Please use a named layer in a `sass_bundle(NAME = layer)`")
+  }
```

Comment:
I think we should make it illegal for bundles or layers to come through any of these inputs